OpenRA.Utility/Program.cs

```diff
@@ -15,6 +15,16 @@
 
 namespace OpenRA.Utility
 {
+	public class NoSuchCommandException : ArgumentException
```

Comment:
Furthermore, ArgumentExceptions are used for method arguments/parameters, not arguments for commands in OpenRA.Utility